test/system/070-build.bats

```diff
@@ -620,6 +620,14 @@ cat >$tmpdir/.dockerignore <<EOF
 *
 EOF
 
+    # Make sure that building a local pause image ignores ignore-files (#13529)
```

Comment:
Because this looks like a NOP (pod create + rm with no actual testing of output), would you mind expanding on the comment? Something like "Prior to this fix, pod-create would fail with 'error building at STEP COPY .../catatonit' because of the ignore pattern". That would save a future maintainer having to look at the issue.